Sources/Fluent/Relations/Children.swift

```diff
@@ -14,7 +14,7 @@ extension Children: QueryRepresentable {
             throw RelationError.noIdentifier
         }
         
-        let foreignId = foreignKey ?? "\(T.name)_\(T.idKey)"
+        let foreignId = foreignKey ?? "\(type(of: parent))_\(T.idKey)".lowercased()
```

Comment:
I don't think this is correct, were you seeing issues w/ `T.name`? I'm wondering if it might be good to have as `foreignKey ?? T.foreignIdKey` as well.